core-service/src/main/java/com/crmfoodestablishment/coreservice/mapper/MenuMapper.java

```diff
@@ -0,0 +1,18 @@
+package com.crmfoodestablishment.coreservice.mapper;
+
+import com.crmfoodestablishment.coreservice.dto.MenuDto;
+import com.crmfoodestablishment.coreservice.entity.Menu;
+import org.mapstruct.*;
+import java.util.UUID;
+
+@Mapper(
+        unmappedTargetPolicy = ReportingPolicy.IGNORE
+)
+public interface MenuMapper {
+    MenuDto mapMenuToMenuDto(Menu menu);
+    Menu mapMenuDtoToMenu(MenuDto menuDto);
+
+    default void setUuid(Menu menu) {
```

Comment:
nope, this logic should be a part of mapMenuDtoToMenu
u can use expression for that
https://www.baeldung.com/java-mapstruct-bean-types-conditional

Reply:
Changed, thank you very much for the link.